app/routes/application.js

```diff
@@ -21,15 +21,20 @@ export default Route.extend(ApplicationRouteMixin, {
 
   model() {
     if (this.get('session.isAuthenticated')) {
-      return this.get('authManager.currentUser').query('notifications', {
-        filter: [
-          {
-            name : 'is-read',
-            op   : 'eq',
-            val  : false
-          }
-        ],
-        sort: '-received-at'
+      return RSVP.hash({
+        notifications: this.get('authManager.currentUser').query('notifications', {
+          filter: [
+            {
+              name : 'is-read',
+              op   : 'eq',
+              val  : false
+            }
+          ],
+          sort: '-received-at'
+        }),
+        pages: this.get('store').query('page', {
```

Comment:
@sumedh123 The reason why the links do not show up at the bottom is that we are making the request when the session is authenticated. So when the user is not logged in the links wont show up on the footer. We need to make a request even when the user is not logged in.

Reply:
@geekyd @niranjan94 I tried that, it can be done like this:
```javascript
model() {
    let notifications = [];
    if (this.get('session.isAuthenticated')) {
      notifications = this.get('authManager.currentUser').query('notifications', {
        filter: [
          {
            name : 'is-read',
            op   : 'eq',
            val  : false
          }
        ],
        sort: '-received-at'
      });
    }

    //
    return RSVP.hash({
      notifications : notifications,
      pages         : this.get('store').query('page', {
        sort: 'index'
      })
    });
  },
```
But then, I ll get and error while running tests that:
```
error  Expected property shorthand              object-shorthand

```
Any way to get rid of this?

Reply:
> Any way to get rid of this?

Yes. Use shorthand property notation. Use `notifications` instead of `notifications : notifications`